sdk/go/kubernetes/provider.go

```diff
@@ -95,6 +101,9 @@ type ProviderArgs struct {
 	// BETA FEATURE - If present and set to true, enable server-side diff calculations.
 	// This feature is in developer preview, and is disabled by default.
 	EnableDryRun pulumi.BoolPtrInput
+	// BETA FEATURE - If present and set to true, replace CRDs on update rather than patching.
```

Comment:
Another thought is to perhaps signal the replacement behavior through an annotation instead? The advantage there would be that it might be more surgical while the provider level configuration is global and will apply to all CRD resources. As discussed offline, we can consider as a subsequent feature based on user feedback.